chart/kubeapps/README.md

```diff
@@ -397,6 +397,8 @@ Once you have installed Kubeapps follow the [Getting Started Guide](https://gith
 | `authProxy.clientID`                              | OAuth Client ID                                                               | `""`                   |
 | `authProxy.clientSecret`                          | OAuth Client secret                                                           | `""`                   |
 | `authProxy.cookieSecret`                          | Secret used by oauth2-proxy to encrypt any credentials                        | `""`                   |
+| `authProxy.cookieRefresh`                         | Duration after which to refresh the cookie                                    | `2m`                   |
```

Comment:
Note these changes are automatically generated using the redminator tool: https://github.com/bitnami-labs/readme-generator-for-helm
(perhaps we should mention it in our guides)

Reply:
Which guide? How's it relevant to users? Sounds like info that should be included at https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md ?